appcompat-theme/src/main/java/dev/chrisbanes/accompanist/appcompattheme/AppCompatTheme.kt

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("AppCompatTheme")
+package dev.chrisbanes.accompanist.appcompattheme
+
+import android.content.Context
+import androidx.compose.material.Colors
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Shapes
+import androidx.compose.material.Typography
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.AmbientContext
+import androidx.core.content.res.use
+
+/**
+ * This function creates the components of a [androidx.compose.material.MaterialTheme], reading the
+ * values from the `Theme.AppCompat` Android theme.
+ *
+ * @param context The context to read the theme from.
+ * @param readColors whether the read the MDC color palette from the context's theme.
+ * If `false`, the current value of [MaterialTheme.colors] is preserved.
+ * @param readTypography whether the read the MDC text appearances from [context]'s theme.
+ * If `false`, the current value of [MaterialTheme.typography] is preserved.
+ * @param shapes A set of shapes to be used by the components in this hierarchy.
+ */
+@Composable
+fun AppCompatTheme(
+    context: Context = AmbientContext.current,
+    readColors: Boolean = true,
+    readTypography: Boolean = true,
+    shapes: Shapes = MaterialTheme.shapes,
+    content: @Composable () -> Unit
+) {
+    val themeParams = remember(context.theme) {
```

Comment:
Any reason why this was skipped?
https://github.com/material-components/material-components-android-compose-theme-adapter/blob/develop/lib/src/main/java/com/google/android/material/composethemeadapter/MdcTheme.kt#L315

Reply:
I was chatting with Adam P about that code a while back. It doesn't really do much right now, since the functions are grey-listed and no-op. Better to just rely on identity `===` and simplify the whole thing.

Reply:
Ah okay. Would you suggest doing the same for MDC-Android Compose Theme Adapter?